GVFS/GVFS.Installer.Windows/Setup.iss

```diff
@@ -763,6 +776,7 @@ begin
       begin
         if ExpandConstant('{param:REMOUNTREPOS|true}') = 'true' then
           begin
+            MigrageVFSForGitConfigFile();

```

Comment:
Should this be moved outside of the `'{param:REMOUNTREPOS|true}'` check?

On a related note, as best I can tell `REMOUNTREPOS` is false when upgrading from GitHub:

https://github.com/microsoft/VFSForGit/blob/4736473e8fc5c1703bb011ad314faf85a14a624b/GVFS/GVFS.Common/GitHubUpgrader.cs#L22

Do you remember why?

Reply:
>>Should this be moved outside of the '{param:REMOUNTREPOS|true}' check?

Yes. Also moved the code that launches `Service UI` to outside this block.

>>Do you remember why?

Yes, b'coz Git upgrader programmatically un-mounts before install and re-mounts after install.

Reply:
Yes - b'coz Github installer programmatically un-mounts before install (so that it can run Git installer before running `VFSForGit` installer) and re-mounts after install.